protocols/CoverageCalculations.sh

```diff
@@ -133,8 +132,9 @@ then
 				percentage=$(echo $((count*100/totalcount)))
 				if [ ${percentage%%.*} -gt 10 ]
 				then
+					${sampleNameID}
```

Comment:
Een variable zonder commando ervoor gaat bash als commando aanroepen en dat gaat hier niet lukken: ik verwacht een error in de vorm van `<waarde van ${sampleNameID}> command not found.`

Reply:
nope, stupid typo  :/